pkg/sshkeys.go

```diff
@@ -20,6 +20,9 @@ func NewSSHKeys() *SSHKeys {
 	sshKeysDir := filepath.Join(homedir, ".ssh")
 	files, err := os.ReadDir(sshKeysDir)
 	if err != nil {
+		if strings.Contains(err.Error(), ".ssh: no such file or directory") {
+			return &SSHKeys{dir: "", keys: nil}
+		}
 		panic(err)
```

Comment:
We should just log this as error and return `&SSHKeys{keys:[]string{}}`

Reply:
yeah this gets logged to the cli we already have that logic in cli.go file

Reply:
<img width="1565" alt="Screenshot 2023-10-25 at 11 03 17 PM" src="https://github.com/nurdsoft/jumpstart/assets/43280610/63f1e52d-e60a-4e6b-84a6-07871d36da36">
here is how it works now @euforia